### ISSUE TITLE
Blank Screen on MacOS 15 (Sequoia) Public Beta

### ISSUE THREAD

**Issue by Leethal88** [rileytestut/Delta#433]
I'm running the public beta for MacOS Sequoia and Delta loads all games with a black screen. The sound is there, but no inputs I have programmed work.

I know that MacOS 15 isn't "officially" out yet, but I wanted to note this for good measure, so it can possibly be looked into ahead of the official release in a couple of months.

Thank you!

**Comment by Butterscotch96** [comment]
Same here

**Comment by Leethal88** [comment]
Yes! I just tested again and N64 works great, but not the other systems! Hoping this gets sorted so I can get back to gaming!

**Comment by a11772553** [comment]
same above.

**Comment by bobbyl140** [comment]
+1

**Comment by Leethal88** [comment]
Looks like there was an update today that fixed it! Make sure to update the app to use it on your Mac again!!!
